ui/app/actions/ProcActions.js

```diff
@@ -17,12 +17,12 @@ export function getProcsFailureAction (error, req) {
   return { type: GET_PROCS_FAILURE, error, req };
 }
 
-export default function fetchProcsAction ({ app, cluster, query }) {
+export default function fetchProcsAction(app, cluster, prefix) {
```

Comment:
Maybe all previous app,cluster,proc listing related methods, filenames should be renamed and fully qualified with profile keyword.
For example, this file becomes ProfileProcActions.js, import variable for this profile indicating the same, etc

Reply:
Made old app,cluster, proc based names fully qualified.